File: src/SOP8/Egovframework/Base/base-model/src/main/java/egovframework/base/model/sop/component/TransmissionTemporary.java
package egovframework.base.model.sop.component;

import lombok.*;
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class TransmissionTemporary {
	public enum Fields { compn_sn, tmpr_sn }

	private int compn_sn;

	private int tmpr_sn;

	public static final String autoIncreaseField = null;
	public static final String tableName = "so_compn_trnsmis_tmpr";
	public static final String[] primaryKeys = { "compn_sn", "tmpr_sn" };

}

File: src/SOP8/Base/Base.Model/Sop/Component/TransmissionTemporary.cs
using System;
using dnsDapperDBUtil.DataAccessLayer.IDAL;

namespace Base.Model.Sop.Component
{
	public class TransmissionTemporary : Table
	{
		public enum Fields { compn_sn, tmpr_sn };
		public enum WriteFields { compn_sn, tmpr_sn };

		public int compn_sn { get; set; }
		public int tmpr_sn { get; set; }

		public static string TableName { get { return "so_compn_trnsmis_tmpr"; } }

		public override string GetTableName()
		{
			return TableName;
		}

		public override string GetPrimaryCondition()
		{
			return string.Format("{0} = {1} and {2} = {3}", Fields.compn_sn, compn_sn, Fields.tmpr_sn, tmpr_sn);
		}

		public override Type GetFieldType()
		{
			return typeof(Fields);
		}

		public override Type GetWriteFieldType()
		{
			return typeof(WriteFields);
		}

		public void FromCopy(TransmissionTemporary obj)
		{
			this.compn_sn = obj.compn_sn;
			this.tmpr_sn = obj.tmpr_sn;
		}
	}
}
